### ISSUE TITLE
Could hosting documentation be expanded and moved to an earlier chapter

### ISSUE THREAD

**Issue by UnclAlDeveloper** [rstudio/plumber#545]
The purpose of an API is to be available to the world.  So hosting should be the second chapter after starting a basic introduction.  And could it be a lot more generic.  Basically it should be what you need to do if you want to host it on your own private server, or AWS, Azure etc., as well as RStudio Connect, Docker and OceanCloud.   I assume it is similar in most of the first cases.  The documentation doesn't even mention that from RStudio $run(host = "0.0.0.0"...) will allow access from the outside world, but that that doesn't work outside of RStudio (and why?).  

That said, I appreciate the chapters on security measures and the various (non-IP) attacks that can occur, and they are very serious considerations.

**Comment by meztez** [comment]
This seems like the same grievance as your other issue, could we combine them and mark them as improve deployment documentation? We currently have some examples in /inst/hosted but it could be expanded.

**Comment by UnclAlDeveloper** [comment]
Hi MezTez,

I am a bit embarrassed by what I posted Friday night (I live in Europe).  I had had a very long and frustrating day when I added the issues (not related to plumber), and was impolite and rude.  Would it be possible to delete / close them?

I didn’t realize the documentation was open source.  Most of the R libraries I follow are fairly restricted in who can modify them.  If I modify the documentation would it be ok to place, a little after the initial examples, something along the lines that “By default plumber is only available to the localhost (127.0.0.1) for testing purposes.  To make it available to the outside world, host = “0.0.0.0”, port = 8000 can be used as parameters to the _run_ function.  This will open up the API to the world.  In terms of security it is recommended to read chapter 7 of the documentation before doing this.”
